### ISSUE TITLE
Wait condition timeout encountered when instantiating cluster template.

### ISSUE THREAD

**Issue by vishwanathj** [F5Networks/f5-openstack-hot#23]
I was unable to re-open the issue #21 , hence opening a new issue. After having specified 2 valid trial license keys in the heat environment file, I still see the original issue reported
![screen shot 2018-06-27 at 1 04 22 pm](https://user-images.githubusercontent.com/7134678/41996941-4cd0b13a-7a0b-11e8-9278-aa39ec085ae2.png)

I also noticed the below peculiar behavior;  would appreciate your comments, if any.
1. Behavior One: When I DO NOT apply valid trial license keys, the two BIG IPs are configured with on being in ACTIVE mode and the other in STANDBY mode. It is  be noted that the stack status is "Create Failed". I logged into the GUI of both the BIG IPs and when I applied the license, both go to Active Active state.
2. Behavior Two: When I DO apply valid trial license keys, the two BIG IPs are configure with both being in ACTIVE Mode. It is  be noted that the stack status is "Create Failed".

**Comment by kaustriaf5** [comment]
@vishwanathj please provide the following for both scenarios. Please mask the license keys. 
1. `openstack stack show {{stackName}}` 
2. contents of `onboard.log`
3. contents of `onboard-cluster.log`
4. contents of `runScript.log`

Also - are you logging in the GUI when the stack is completely done?

**Comment by vishwanathj** [comment]
The output of openstack stack show is at http://paste.openstack.org/show/724452/

Contents of onboard.log for BigIpI: http://paste.openstack.org/show/724454/
Contents of onboard-cluster.log for BigIp1: http://paste.openstack.org/show/724455/
Contents of runScript.log for BigIp1:  http://paste.openstack.org/show/724456/

Contents of onboard.log for BigIp2: http://paste.openstack.org/show/724460/
Contents of onboard-cluster.log for BigIp2: http://paste.openstack.org/show/724461/
Contents of runScript.log for BigIp2: http://paste.openstack.org/show/724462/

The outputs for scenario i.e Behavior One was attached to issue #21 ; however, that did not have the onboard-cluster.log and that stack has been deleted.

**Comment by vishwanathj** [comment]
Yes, I am able to login into the GUI of the BIG IP after the stack fails.

**Comment by kaustriaf5** [comment]
After working with @vishwanathj, determined this was comprised of 2 issues:
1. Heat Signal showing onboard error could not be sent due to invalid json included in the error message (fixed in https://github.com/F5Networks/f5-openstack-hot/tree/v4.0.1)
2. Licensing request times out/400 Bad Request on their environment (pending)
